hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/recon/TestReconWithOzoneManagerHA.java

```diff
@@ -128,7 +124,7 @@ public void testReconGetsSnapshotFromLeader() throws Exception {
         ozoneManager.get().getHttpServer().getHttpAddress().getPort() +
         OZONE_DB_CHECKPOINT_HTTP_ENDPOINT;
     String snapshotUrl = impl.getOzoneManagerSnapshotUrl();
-    Assert.assertEquals("OM Snapshot should be requested from the leader.",
+    assertEquals("OM Snapshot should be requested from the leader.",
         expectedUrl, snapshotUrl);
```

Comment:
JUnit5 assertions accept failure message as last parameter instead of first one.  So here `"OM Snapshot ..."` is used as `expected` value, which causes test failure:

```
AssertionFailedError: http://localhost:15005/dbCheckpoint ==> expected: <OM Snapshot should be requested from the leader.> but was: <http://localhost:15005/dbCheckpoint>
	at org.junit.jupiter.api.AssertionFailureBuilder.build(AssertionFailureBuilder.java:151)
	at org.junit.jupiter.api.AssertionFailureBuilder.buildAndThrow(AssertionFailureBuilder.java:132)
	at org.junit.jupiter.api.AssertEquals.failNotEqual(AssertEquals.java:197)
	at org.junit.jupiter.api.AssertEquals.assertEquals(AssertEquals.java:182)
	at org.junit.jupiter.api.Assertions.assertEquals(Assertions.java:1156)
	at org.apache.hadoop.ozone.recon.TestReconWithOzoneManagerHA.testReconGetsSnapshotFromLeader(TestReconWithOzoneManagerHA.java:127)
```

Message should be last argument (or can be dropped completely).

```suggestion
    assertEquals(expectedUrl, snapshotUrl);
```